CSharp/Library/Microsoft.Bot.Connector.Shared/JwtConfig.cs

```diff
@@ -9,9 +9,12 @@
     public static class JwtConfig
     {
         /// <summary>
-        /// TO CHANNEL FROM BOT: Login URL
+        /// TO CHANNEL FROM BOT: Login URL format string. Bot developer can optionally specify
+        /// which tenant to obtain an access token from. The default tenant is "botframework.com".
+        /// Normally the chanels only accept tokens from the default tenant.
+        /// TODO: add doc link
         /// </summary>
-        public const string ToChannelFromBotLoginUrl = "https://login.microsoftonline.com/botframework.com/oauth2/v2.0/token";
+        public const string ToChannelFromBotLoginUrlFormat = "https://login.microsoftonline.com/{0}/oauth2/v2.0/token";

```

Comment:
I would consider naming this "ToChannelFromBotLoginUrlTemplate", as "Template" is a much better name than "Format". The template name could be re-used across JS/Python/Java, whereas the Format (as in "String.Format") is unique to C#.

Reply:
Note that "URL Template" is an industry standard term (RFC's and everything!):
https://en.wikipedia.org/wiki/URL_Template

Reply:
yep. will do.